cmd/minishift/cmd/logs.go

```diff
@@ -48,6 +49,7 @@ var logsCmd = &cobra.Command{
 }
 
 func init() {
-	logsCmd.Flags().BoolVarP(&follow, "follow", "f", false, "Continously print the logs entries.")
+	logsCmd.Flags().BoolVarP(&follow, "follow", "f", false, "Continuously print the logs entries")
+	logsCmd.Flags().Int64VarP(&tail, "tail", "t", -1, "Number of lines to show from the end of the logs")
```

Comment:
:+1:

Reply:
@budhram Does this mean ` $ minishift logs -t` will work with default value -1?

```
 $ minishift logs -t
Error: flag needs an argument: 't' in -t
Usage:
  minishift logs [flags]

Flags:
  -f, --follow     Continuously print the logs entries
  -h, --help       help for logs
  -t, --tail int   Number of lines to show from the end of the logs (default -1)
Global Flags:
      --alsologtostderr                  log to standard error as well as files
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory (default "")
      --logtostderr                      log to standard error instead of files
      --profile string                   Profile name (default "minishift")
      --show-libmachine-logs             Show logs from libmachine.
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          log level for V logs. Level varies from 1 to 5 (default 1).
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

flag needs an argument: 't' in -t
```

Reply:
docker do require an argument but the unix command tail(1) by default do 10 :accoding to its [manpage](http://man7.org/linux/man-pages/man1/tail.1.html)